src/server/game/World/World.cpp

```diff
@@ -3684,6 +3747,98 @@ void World::RemoveOldCorpses()
     m_timers[WUPDATE_CORPSES].SetCurrent(m_timers[WUPDATE_CORPSES].GetInterval());
 }
 
+uint8 GetFactionOutnumberedRewardEffectValue(FactionOutnumberReward reward)
+{
+    uint8 baseValue = 10;
+    switch (reward)
+    {
+        case FactionOutnumberReward::Overwhelming:
+        case FactionOutnumberReward::Percent20: return baseValue + 20;
```

Comment:
where does this 20% come from? "overwhelming" is equal to 5% on retail currently.
More specific (EU):
Horde: 10% (normal)
Alliance: 15% (has overwhelming odds quest)

Reply:
From various sources online. The source can be *up* to 20%, depending on the amount.
For example:
https://wow.gamepedia.com/War_Mode ` If a faction is outnumbered, a War Mode: Call to Arms adds up to a further +20% bonus.`

https://www.reddit.com/r/wow/comments/kq0nj5/war_mode_enlisted_bonuses/gi34b4f/?utm_source=reddit&utm_medium=web2x&context=3
I guess it's a lot more rare since the percentage is higher, but I believe this is how it's done.

You disagree?

Reply:
That wiki has wrong info. Call to Arms bonus scales in steps of 5%

Reply:
@Shauren I don't think it says otherwise, and that's in fact what I think it is. Although I don't think a total of 25% bonus exist; I haven't seen one anyway.

@mdX7 I believe I misread what you wrote.
At first I thought you meant there shouldn't be any 20% extra, and the maximum should be 10% (total 20%)).
Reading again, I now understand it means:
1. 5% extra (15% total)
2. 10% extra (20% total) + overwhelming
3. 20% extra (30% total)

The part about overwhelming being 20% and not 10% was a complete guess and not from a soruce.
So yeah I will change it.